generators/app/templates/src/environments/environment.prod.ts

```diff
@@ -1,4 +1,9 @@
 // `.env.ts` is generated by the `npm run env` command
+// Using `npm run env` is to expose environment variables as JSON for any usage you might
```

Comment:
"Using `npm run env` is to expose" -> "`npm run env` expose"

makes it easier to read IMO

Reply:
cool beans :) i'll reword it tonight after I eat something ;)

Reply:
updated